src/Resources/Resources.Test/ScenarioTests/DeploymentScriptsTests.cs

```diff
@@ -18,7 +18,7 @@
 
 namespace Microsoft.Azure.Commands.Resources.Test.ScenarioTests
 {
-    public class DeploymentScriptsTests : ResourceTestRunner
+    public class DeploymentScriptsTests : ResourcesTestRunner
```

Comment:
I recommend that your don't need to rename it even it doesn't follow convention. It can reduce the size of your pull request.

Reply:
Currently the naming convention for the TestRunner file is {Test-Project-Name} + TestRunner. We have almost all the test projects that comply with this standard except for this one. In order to make all the test runner names consistent within the whole repo, I strongly believe this change is necessary. I concede it would reduce the size of this PR. But since it is necessary, better we can keep the class name as it is.